yoeo/utils/loss.py

```diff
@@ -122,9 +122,9 @@ def compute_loss(combined_predictions, combined_targets, model):
         lobj += BCEobj(layer_predictions[..., 4], tobj) # obj loss
 
     # Scalaing of losses
-    lbox *= 0.2
+    lbox *= 0.05
     lobj *= 1.0
-    lcls *= 0.05
+    lcls *= 0.5
```

Comment:
Is this intentional?

Reply:
This was the result of some hyperparameter optimization in YOLO, but I also thought about droping it in the merge as we might need to test them first to see how they affect YOEO.